test-resources/sonar-project.properties

```diff
@@ -0,0 +1,9 @@
+sonar.projectKey=mobile-id-check-async-test-resources
```

Comment:
The project was only created 45 mins ago, I'm impressed you found this as I hadn't told you yet :D

Reply:
I'm not sure what's more impressive, finding the key or making a best guess at what it could be and getting it right... maybe it's best to not admit to either 😅

Reply:
It's a sign we both chose the best name!